movie/src/App.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Footer from "./components/footer/Footer";
```

Comment:
You should use case with first capital letter and then camelCase for each component folder.
Also create index file with import and export.
Then your path becomes into "./components/Footer" instead of your.

Reply:
update